docs/cel2/decision-tree.md

```diff
@@ -21,10 +21,11 @@ description: Not sure what you need to do to migrate from Celo L1 to Celo L2?
 
   If none of the above work for you, you can use cLabs best effort hosted node: [Forno](https://docs.celo.org/network/node/forno).
 </details>
+</br>
 <details>
   <summary>I use ContractKit or Fee Abstraction (ERC-20 gas tokens)</summary>
 
-  Recommended versions for SDK and libraries:
+  **Required** versions for SDK and libraries:
```

Comment:
Technically not true, see https://github.com/celo-org/docs/pull/1465#discussion_r1768042502, but fine by me if that can get people to upgrade.